test/e2e-test/suite_test.go

```diff
@@ -94,6 +94,15 @@ var _ = BeforeSuite(func(done Done) {
 	}
 	By("Finished setting up test environment")
 
+	By("Applying CRD of WorkloadDefinition and TraitDefinition")
+	var workloadDefinitionCRD crdv1.CustomResourceDefinition
+	Expect(readYaml("../../charts/oam-kubernetes-runtime/crds/core.oam.dev_workloaddefinitions.yaml", &workloadDefinitionCRD)).Should(BeNil())
+	Expect(k8sClient.Create(context.Background(), &workloadDefinitionCRD)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
+
+	var traitDefinitionCRD crdv1.CustomResourceDefinition
+	Expect(readYaml("../../charts/oam-kubernetes-runtime/crds/core.oam.dev_traitdefinitions.yaml", &traitDefinitionCRD)).Should(BeNil())
+	Expect(k8sClient.Create(context.Background(), &traitDefinitionCRD)).Should(SatisfyAny(BeNil(), &util.AlreadyExistMatcher{}))
+
```

Comment:
The applying of those two CRDs will just add several seconds (3-5 seconds) for CI/e2e-test stage, but it will help save much time in local development.

Reply:
take a look at https://github.com/crossplane/oam-kubernetes-runtime/blob/master/pkg/controller/v1alpha2/applicationconfiguration/suite_test.go#L57-L61

Reply:
Great, let me use the built-in `InstallCRDs` function from controller-runtime instead.